submissions/evgeniy24/js-dom/script.js

```diff
@@ -0,0 +1,129 @@
+const container = document.querySelector(".container");
+const nav = document.querySelector(".menu");
+const content = document.createElement("div");
+
+function createMainDescription(elem) {
+  nav.after(elem);
+  elem.classList.add("description");
+
+  const descriptionHeader = document.createElement("h2");
+  descriptionHeader.classList.add("description__header");
+  descriptionHeader.textContent = "Star Wars";
+  elem.appendChild(descriptionHeader);
+
+  const divForImg = document.createElement("div");
+  divForImg.classList.add("description__img-container");
+  elem.appendChild(divForImg);
+
+  const img = document.createElement("img");
+  img.setAttribute("src", "images/starWarsSaga.jpg");
+  img.setAttribute("alt", "star wars");
+  divForImg.appendChild(img);
+
+  const divForText = document.createElement("div");
+  divForText.classList.add("description__text-container");
+  elem.appendChild(divForText);
+
+  const paragForText = document.createElement("p");
+  paragForText.textContent =
+    "Star Wars is an American epic space-opera media franchise created by George Lucas, which began with the eponymous 1977 film and quickly became a worldwide pop-culture phenomenon. The franchise has been expanded into various films and other media, including television series, video games, novels, comic books, theme park attractions, and themed areas, comprising an all-encompassing fictional universe.";
+  divForText.append(paragForText);
+}
+createMainDescription(content);
+
+function filmPatter(filmList, item) {
+  const filmHeader = item.querySelector("h2");
+  filmHeader.textContent = filmList.header;
+
+  const filmImage = item.querySelector("img");
+  filmImage.setAttribute("src", filmList.image);
+  filmImage.setAttribute("alt", filmList.alt);
+
+  const filmText = item.querySelector("p");
+  filmText.textContent = filmList.text;
+}
+
+nav.addEventListener("click", filmGenerator);
+
+function filmGenerator({ target }) {
```

Comment:
Maybe it will be better to use function expressions instead of function declarations

Reply:
Fixed - now it uses function expressions